PeakLocalisations/Localisation/App Store/Screenshots.xcstrings

```diff
@@ -168,6 +222,12 @@
             "state" : "translated",
             "value" : "...And For Your Lockscreen"
           }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "...Y Para Su Pantalla de Bloqueo"
```

Comment:
'pantalla de bloqueo' sounds off to me, is this the apple term?

Reply:
Do you think "Pantalla bloqueada" works better? I've seen both spellings

Reply:
I prefer Pantalla de bloqueo